client/components/select-dropdown/test/index.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+import { assert } from 'chai';
+import { shallow, mount } from 'enzyme';
+import sinon from 'sinon';
```

Comment:
> I was wondering why useSandbox does not just return the newly created sandbox;

I'm not quite sure now, but most likely we wanted to have one style of using test helpers. It was the only helper that was returning a value. Feel free to open PR if you have an idea how to improve our code. 👍